exploratory-tests/tesztterv.md

```diff
@@ -0,0 +1,126 @@
+Test plan
+
+Unless explicitly written you can reproduce these tests with emulator or real, physical device.
+
+## 1. test
+Test title: Google sign-in testing.\
+Test objective: Testing wether the user is able to log in with his/her Google account.\
+Test steps: 
+- Start the application
+- Click-on Google Sign-in in the middle of the screen.
+Test result: Successful
+
+## 2. test
+Test title: Tracking service testing between 2 near locations.\
+Test objective: Testing wether we can start the service and the service sends location information accordingly\
+Test steps:
+- Log-in with Google-account
+- Start tracking service (when on real physical device, if you are using emulator set route to be between 2 locations within Budapest and start simulating). 
+- Click on sessions in the bottom right corner.
+- Scroll to the last session (this is the one you just started).
+- Click on the map icon on the right (it is important to click the map icon and not something else on the same session line, because that will take you to the session's infomation.)
+    ![Zoomolás szimulálása](2latszik_ahogy_halad_mindig_helyesen_van_bezoomolva.png)\
```

Comment:
A képek egy részének .**PNG** a kiterjesztése, ami miatt például ez a kép nem is jelenik meg githubon. Át tudnád légyszi nevezni, hogy mindehol egységesen .**png** legyen a kiterjesztés?

Reply:
Ez még mindig rossz

Reply:
Én az <img> taget használtam a képeimnél. Furcsa egyébként az a dangling \ jel is a végén... El tudod távolítani @Adinator007 ?